Process descendant filtering in event filters [serverless] (#5672)
elastic/security-docs: docs/serverless/edr-manage/event-filters.mdx

      1. `Description`: Enter a filter description (optional).
1. In the **Conditions** section, depending which page you're using to create the filter, either modify the pre-populated conditions or add new conditions to define how ((elastic-sec)) will filter events. Use these settings:
    1. `Select operating system`: Select the appropriate operating system.
    1. Select which kind of event filter you'd like to create:
    
        * `Events`: Create a generic event filter that can match any event type. All matching events are excluded.

>         * `Process Descendants`: Create a filter that suppresses the descendant activity of a specified process. Events from the matched process will be ingested, but events from its descendant processes will be excluded.

Author: Slight tweak to start the sentence with an active command:
```suggestion
        * `Process Descendants`: Specify a process, and suppress the activity of its descendant processes. Events from the matched process will be ingested, but events from its descendant processes will be excluded.
```

>         * `Process Descendants`: Create a filter that suppresses the descendant activity of a specified process. Events from the matched process will be ingested, but events from its descendant processes will be excluded.

Reviewer: Oh this is even better!
